Dockerfile

```
@@ -0,0 +1,13 @@
+FROM golang:1.22.0
+
+WORKDIR /app
+
+COPY go.mod go.sum tracker.db ./
```

БД лучше копировать в 9 строчке, так как если хоть что-то в БД поменяется, то заново будут качаться все зависимости

```suggestion
COPY go.mod go.sum ./
```

---

Принято! Спасибо